index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <div class="hero">
+            <nav class="navbar">
+                <h2 class="logo">Cobby Elson</h2>
+                <button type="button" class="hamburger-btn">
+                    <span class="toggler-icon top-bar"></span>
+                    <span class="toggler-icon middle-bar"></span>
+                    <span class="toggler-icon bottom-bar"></span>
+                </button>
+            </nav>
```

Comment:
Kindly implement these changes for the toolbar section to have a better UI and a better user experience :

-  Your logo on line 17 should be in an `<a href =""></a>` tag and not an `h2` tag for good semantics as it is shown in your code 😄 
-  The hamburger button should be in a `nav` tag for semantics reasons, and the right position for convention and user experience for mobile devices

Reply:
@usorfaitheloho  Thanks  please,  but with the second comment thus 

- The hamburger button should be in a nav tag for semantics reasons, and the right position for convention and user experience for mobile devices
- The logo and button are already in a nav tag, so should i strip that one off and use a div  in place of that whiles I use a nav tag only for the button?